trino/trino.go

```diff
@@ -101,18 +101,19 @@ var (
 const (
 	preparedStatementHeader    = "X-Trino-Prepared-Statement"
 	preparedStatementName      = "_trino_go"
-	trinoUserHeader            = "X-Trino-User"
-	trinoSourceHeader          = "X-Trino-Source"
-	trinoCatalogHeader         = "X-Trino-Catalog"
-	trinoSchemaHeader          = "X-Trino-Schema"
-	trinoSessionHeader         = "X-Trino-Session"
-	trinoSetCatalogHeader      = "X-Trino-Set-Catalog"
-	trinoSetSchemaHeader       = "X-Trino-Set-Schema"
-	trinoSetPathHeader         = "X-Trino-Set-Path"
-	trinoSetSessionHeader      = "X-Trino-Set-Session"
-	trinoClearSessionHeader    = "X-Trino-Clear-Session"
-	trinoSetRoleHeader         = "X-Trino-Set-Role"
-	trinoExtraCredentialHeader = "X-Trino-Extra-Credential"
+	trinoHeaderPrefix = `X-Trino-`
+	trinoUserHeader            = trinoHeaderPrefix+`User`
+	trinoSourceHeader          = trinoHeaderPrefix+`Source`
+	trinoCatalogHeader         = trinoHeaderPrefix+`Catalog`
+	trinoSchemaHeader          = trinoHeaderPrefix+`Schema`
+	trinoSessionHeader         = trinoHeaderPrefix+`Session`
+	trinoSetCatalogHeader      = trinoHeaderPrefix+`Set-Catalog`
+	trinoSetSchemaHeader       = trinoHeaderPrefix+`Set-Schema`
+	trinoSetPathHeader         = trinoHeaderPrefix+`Set-Path`
+	trinoSetSessionHeader      = trinoHeaderPrefix+`Set-Session`
+	trinoClearSessionHeader    = trinoHeaderPrefix+`Clear-Session`
+	trinoSetRoleHeader         = trinoHeaderPrefix+`Set-Role`
+	trinoExtraCredentialHeader = trinoHeaderPrefix+`Extra-Credential`
```

Comment:
These change reminds me https://github.com/trinodb/trino-python-client/pull/28. I prefer the original values.

Reply:
sure! i don't like how it looks either, although i made this change because if header prefix and headers themselves are separate variables then there is a risk that only one will be changed.